tests/test_utilities.py

```diff
@@ -1149,6 +1149,23 @@ def test_get_course_run_start_with_default(self):
         assert utils.get_course_run_start({}) is None
         assert utils.get_course_run_start({}, now) == now
 
+    @ddt.data(
+        (
+            fake_catalog_api.create_course_run_dict(end="2030-10-15T13:11:03Z", weeks_to_complete=1000),
```

Comment:
nit: Can we avoid using hard coded dates for tests data? May be something like `end=datetime.utcnow() + timedelta(days=20)`